README.rst

```diff
@@ -166,6 +166,42 @@ Let's install ``beartype`` with Anaconda_, because corporate standards are
    conda config --add channels conda-forge
    conda install beartype
 
+If you're feeling the love and wanting to celebrate, consider signaling that you're bear-ified:
+
+  YummySoft is now |bear-ified|!
+
+.. |bear-ified| image:: https://raw.githubusercontent.com/posita/beartype-assets/postita/0/add-bear-ified-badge/badge/bear-ified.svg
```

Comment:
This needs to be changed to `https://raw.githubusercontent.com/beartype/beartype-assets/main/badge/bear-ified.svg`. I put this hear temporarily to enable rending until beartype/beartype-assets#4 is merged.

Reply:
Should be fixed now.